.github/workflows/status-checks.yml

```diff
@@ -8,47 +8,52 @@ on:
     branches: [ master ]
 
 jobs:
-      
+
   lint:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
 
     - name: Set up Go
-      uses: actions/setup-go@v3
+      uses: actions/setup-go@v4
       with:
-        go-version: ^1.18
+        go-version-file: go.mod
```

Comment:
I think this makes that the lint fails at the moment. Can you revert this to using ^1.18 and above? Or ^1.21 is also fine.